Katello/katello
# Fixes #35446 - Change content source improvements

#### What are the changes introduced in this pull request?
Follow up to https://github.com/Katello/katello/pull/9873

Bug fixes:
* Select all hosts from host index (not ignoring `search` parameter anymore)
* Load proxies with content AND environment

Improvements
* Select proxy first, avoid mismatch of environment, content view and capsule
* Lifecycle environment in '/katello/api/capsule'
* Better UI for list of hosts with search input

#### Considerations taken when implementing this change?
* Discussion and notes from the CEE demo

#### What are the testing steps for this pull request?
* Select one or more hosts (with content & without content)
* Go to the form
* Verify that displaying hosts works (to update & ignored)
* Check loaded data in the form - now it shouldn't be possible to select incorrect proxy / environment / content view
* Change the content source & verify that hosts consume content from new endpoint
